.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
What I do is use global gitignore for this. http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

Reply:
Do you have any recommendations for values to add here besides '.DS_Store'?

Reply:
Anything system-specific goes in the global gitignore; on macOS I mostly know about DS_Store. (On windows Thumbs.db is a popular one)

Anything project-specific goes in the project gitignore. At this stage I don't think you have anything, so you can remove this file once you have a system one set up.